spec/paths/v1@enterprise@{enterprise_id}@.yaml

```diff
@@ -9,7 +9,7 @@ get:
     '200':
       description: successful operation
       schema:
-        $ref: '#/definitions/Enterprise'
+        $ref: '#/definitions/EnterpriseV1'
```

Comment:
Given that we are deleting Enterprise definition. W can rename this to Enterprise only right ? Only the path needs to be changed to v1. Let me know your thoughts.

Reply:
have created separated folders in definitions for V1 definitions.